airqo-platform/AirQo-api
# Ft device incident reports

- What does this PR do?
This PR creates endpoints which can be used for the various components of the monitoring dashboard
- What JIRA task is related to this PR?
I cannot access the link right now.... 😲
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/data-mgt` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the end points using any REST client tool available to you. `Redis DB` is being utilized here for caching so please make sure that you have it running on your local machine.
- Which end points should are ready for testing?
> - partial url: /api/v1/data/channels/age?query=CHANNELID&channel=CHANNELID
> - data/channels/fields/age?query=FIELDNAME-CHANNELID&channel=CHANNELID&field=FIELDNAME
>-  api/v1/data/channels/count?query=COUNT

Please access [AirQo API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=757070984) (**data-mgt** sheet) for more details
